taxonomies/labels.txt

```diff
@@ -4198,6 +4259,9 @@ xx:LV-BIO-01
 en:LV-BIO-02
 xx:LV-BIO-02
 
+en:LK-BIO-149
+fr:LK-BIO-149
```

Comment:
```suggestion
xx:LK-BIO-149
```

xx: is like a wildcard, it will match all languages

Reply:
This edit is actually not from me but from the "original" label branch. But sure I will change it.